tests/crud/test_crud_endpoints.py

```diff
@@ -282,6 +279,24 @@ def test_patch_fails(self):
         response = client.patch(f"/{movie.id}/", json={"foo": "bar"})
         self.assertEqual(response.status_code, 400)
 
+    def test_patch_validation_error(self):
+        """
+        Check if Pydantic validation error works.
+        """
+        client = TestClient(PiccoloCRUD(table=Movie, read_only=False))
+
+        movie = Movie(name="Star Wars", rating=93)
+        movie.save().run_sync()
+
+        response = client.patch(
+            f"/{movie.id}/",
+            json={"name": 95, "rating": "95"},
+        )
+        self.assertIn("validation error", str(response.content))
+        self.assertEqual(response.status_code, 400)
+
+        self.assertEqual(Movie.count().run_sync(), 1)
```

Comment:
I'm going to change this assertion - because we want to make sure the existing row didn't change.